pkgs/lints/tool/gen_docs.dart

```diff
@@ -10,8 +10,10 @@ import 'package:yaml/yaml.dart' as yaml;
 import 'package:http/http.dart' as http;
 
 /// Source of truth for linter rules.
-const rulesUrl =
-    'https://raw.githubusercontent.com/dart-lang/site-www/main/src/_data/linter_rules.json';
+final rulesUri = Uri.https(
+  'raw.githubusercontent.com',
+  'dart-lang/site-www/main/src/data/linter_rules.json',
+);
```

Comment:
Unrelated nit, feel free to ignore, but the comment on line 30 below ends prematurely (or has a hanging "to" that should be deleted).